src/components/ConsortiaControlledVocabulary/ConsortiaControlledVocabulary.js

```diff
@@ -147,9 +164,21 @@ export const ConsortiaControlledVocabulary = ({
     }).finally(setActiveDialog);
   }, [translations]);
 
-  const showSuccessCallout = useCallback(({ actionType, entry, members }) => {
+  const showSuccessCallout = useCallback(({ actionType, entry }) => {
     const translationKey = TRANSLATION_KEYS_MAP[actionType];
 
+    const getLocalTenantName = () => [selectedMembers?.find(({ id: _id }) => entry.tenantId === _id)?.name];
+
+    const membersGetters = {
+      [ACTION_TYPES.create]: () => selectedMembers.map(({ name }) => name),
```

Comment:
Can be changed from `[ACTION_TYPES.create]: () => selectedMembers.map(({ name }) => name),` to 
`[ACTION_TYPES.create]: getSeletedMembers` to be consistent